models/record.go

```diff
@@ -34,12 +34,14 @@ import (
 //     ALIAS
 //     CF_REDIRECT
 //     CF_TEMP_REDIRECT
+//     CF_WORKER_ROUTE
 //     FRAME
 //     IMPORT_TRANSFORM
 //     NAMESERVER
 //     NO_PURGE
 //     NS1_URLFWD
 //     PAGE_RULE
+//     WORKER_ROUTE
```

Comment:
Is there a "WORKER_ROUTE"?

Reply:
Yes. `WORKER_ROUTE` is for `CF_WORKER_ROUTE` what a `PAGE_RULE` is for a `CF_REDIRECT`/`CF_TEMP_REDIRECT`.